controllers/helmchart_controller_test.go

```diff
@@ -406,7 +406,7 @@ var _ = Describe("HelmChartReconciler", func() {
 			By("Expecting artifact")
 			Eventually(func() bool {
 				_ = k8sClient.Get(context.Background(), key, got)
-				return meta.HasReadyCondition(got.Status.Conditions)
+				return meta.InReadyCondition(got.Status.Conditions)
```

Comment:
Taking the other comments into account, this would become:

```suggestion
				return apimeta.IsStatusConditionTrue(got.Status.Conditions, meta.ReadyCondition)
```